### ISSUE TITLE
How to just copy issue not move?

### ISSUE THREAD

**Issue by 345161974** [google/github-issue-mover#133]
Hi, Thanks for the GREAT TOOLS first!

but the problem is I just want to copy issue from repo A to repo B? not move. How to do this? (I don't want to close issue in repo A).

**Comment by Campbell-Jackson** [comment]
Was there an answer?

**Comment by jdx-john** [comment]
Yes all those automated tasks - copying comments, etc, would be better optional. I want to expose copies of issues to a customer-facing issue without all our developer comments viewable.

Is this project still maintained?

**Comment by fgnievinski** [comment]
marketplace alternative:
https://github.com/marketplace/actions/github-issue-copier
